src/Commands/Self/Plugin/CreateCommand.php

```diff
@@ -15,7 +15,8 @@ class CreateCommand extends PluginBaseCommand
 {
     const USAGE_MESSAGE = 'terminus self:plugin:create <path>';
     const EXISTING_FOLDER_MESSAGE = 'Path should be a non-existing folder that will be created';
-    const COMPOSER_CREATE_PROJECT = 'composer create-project pantheon-systems/terminus-plugin-example {dir}';
+    const COMPOSER_CREATE_PROJECT =
```

Comment:
@kporras07 If this is switched temporarily, when do we need to roll this back again? Would it be helpful to add some kind of `todo` comment about the fact that this is a temporary solution?

Reply:
Fixed. Thanks. Merging when tests pass.